index.js

```diff
@@ -784,7 +784,7 @@ const ContainerService = {
       } else {
         this._recentBrowserWindow().then(browserWin => {
           resolve(browserWin);
-        });
+        }).catch(() => {});
```

Comment:
Probably should throw something like:
```
new Error("This shouldn't happen when checking for a window");
```

Or similar.... and any other catch's we have without errors